css/ampshared.css

```diff
@@ -539,6 +539,19 @@ amp-accordion {
   display: block !important;
 }
 
+/* heading */
```

Comment:
Would be good to add a comment for why this stuff is here.

Reply:
Good idea, adding now.

Reply:
Added comment, inline reference here.

```
This media query is used instead of @supports due to low compatibility.
At the time of this change (4/6/2021) no version of Webkit support @supports queries.

These provide low-specificity UI styles to prevent CLS for the header and content 
segments of accordions.
```